scripts/install_px4.sh

```diff
@@ -0,0 +1,28 @@
+#Download and install PX4.
+# Run with: sudo bash ./install_px4.sh
+
+cd ../..
```

Comment:
What purpose does this server? Perhaps it would be cleaner to do a cd into third-party folder within pteranodon itself. LMK if you want to go down that route, I think it could be nicer for the user so they have a single folder with everything in it. (Then we could symlink as well)

Reply:
Are you suggesting we clone PX4 inside of pteranodon? Or include it in our repo?

Reply:
In the third-party branch I have implemented 2 submodules. PX4 would get cloned into the third-party repo if the user pulls with --recurse-submodules option. Then we build PX4 in place inside the repo. 

I am going to table that for now, but make an issue about it if that sounds good to you?